apps/dialtone-documentation/docs/components/keyboard-shortcut.md

```diff
@@ -56,6 +65,89 @@ vueCode='
 '
 showHtmlWarning />
 
+### Shortcut size variation
+
+<code-well-header>
+  <dt-keyboard-shortcut
+    class="d-fs-200"
+    shortcut="{cmd}+Y"
+    ref="sizeExample"
+  />
+</code-well-header>
+
+<code-example-tabs
+:htmlCode="() => $refs.sizeExample"
+vueCode='
+<dt-keyboard-shortcut class="d-fs-200" shortcut="{cmd}+Y" />
+'
+/>
+
+### All available shortcut aliases
+
+<code-well-header>
+  <dt-keyboard-shortcut
+    ref="allShortcuts"
+    shortcut="{cmd}+{win}+{arrow-right}+{arrow-left}+{arrow-up}+{arrow-down}"
+  />
+</code-well-header>
+
+<code-example-tabs
+:htmlCode="() => $refs.allShortcuts"
+vueCode='
+<dt-keyboard-shortcut
+  shortcut="{cmd}+{win}+{arrow-right}+{arrow-left}+{arrow-up}+{arrow-down}"
+/>
+'
+/>
+
+### Inline with text and screen reader text
+
+<code-well-header>
+  <div ref="inlineExample">
+  Press
+  <dt-keyboard-shortcut
+    screen-reader-text="Ctrl and F5"
+    shortcut="Ctrl + F5"
+  />
+  to hard refresh the page.
+  </div>
+</code-well-header>
+
+<code-example-tabs
+:htmlCode="() => $refs.inlineExample"
+vueCode='
+Press
+<dt-keyboard-shortcut
+  screen-reader-text="Ctrl and F5"
+  shortcut="Ctrl + F5"
+/>
+to hard refresh the page.
+'
+/>
```

Comment:
Hmm this one seems to be rendering the screen-reader-text.

<img width="953" alt="Screenshot 2024-08-07 at 1 47 26 PM" src="https://github.com/user-attachments/assets/9cfecff3-2b16-4502-9a46-afbdf1504708">